### PR TITLE
Post list fixes

### PR DESCRIPTION
**Pull request by charlag**

### CODE REVIEW

**Comment by ganthern** [comment]
some repro steps (including the way the bugs manifest) would be nice on the fixed issues because the titles are not entirely clear and there are barely any commit messages

File: src/mail/view/MailViewModel.ts [hunk 1]
```diff
@@ -138,12 +138,13 @@ export class MailViewModel {
...
 		if (this.conversationViewModel && this.conversationPref !== conversationEnabled) {
+			const mail = this.conversationViewModel.primaryMail
 			this.createConversationViewModel({
-				mail: this.conversationViewModel.primaryMail,
+				mail,
 				showFolder: false,
-				// FIXME
 				delayBodyRenderingUntil: Promise.resolve(),
 			})
+			this.mailOpenedListener.onEmailOpened(mail)
```
Comment (ganthern) [inline src/mail/view/MailViewModel.ts:147]: just from the diff it's not clear why moving the callback call out of .createConversationViewModel fixes the multiple calls

**Comment by charlag** [comment]
I've added repro steps, I will also improve commit messages

**Comment by ganthern** [comment]
fixes look OK to me otherwise

**Pull request merged**